src/entities/transactions/transactionType.ts

```diff
@@ -1,37 +1,24 @@
-export enum TransactionType {
-  authorize = 'authorize',
-  borrow = 'borrow',
-  cancel = 'cancel',
-  contract_interaction = 'contract interaction',
-  deployment = 'deployment',
-  deposit = 'deposit',
-  dropped = 'dropped',
-  execution = 'execution',
-  mint = 'mint',
-  purchase = 'purchase', // Rainbow-specific type
-  receive = 'receive',
-  repay = 'repay',
-  sell = 'sell',
-  send = 'send',
-  trade = 'trade',
-  withdraw = 'withdraw',
-}
-
-export default {
-  authorize: 'authorize',
-  borrow: 'borrow',
-  cancel: 'cancel',
-  contract_interaction: 'contract interaction',
-  deployment: 'deployment',
-  deposit: 'deposit',
-  dropped: 'dropped',
-  execution: 'execution',
-  mint: 'mint',
-  purchase: 'purchase',
-  receive: 'receive',
-  repay: 'repay',
-  sell: 'sell',
-  send: 'send',
-  trade: 'trade',
-  withdraw: 'withdraw',
-};
+export const transactionTypes = {
```

Comment:
to match BX. separating allows us to better track if we should be showing asset changes/diffs